.github/workflows/php-code-analysis-legacy.yml

```diff
@@ -0,0 +1,46 @@
+name: PHP
+
+on:
+    workflow_call:
+
+jobs:
+    call-legacy-analyse-workflow:
+        name: Analyse code for SwagPaymentPayPalUnified with legacy PHP version
+        runs-on: ubuntu-latest
+        container:
+            image: ghcr.io/shopware5/docker-images-testing/install:shopware_5.2_5.7_5.6_5.2.11-5.2.27
+            credentials:
+                username: ${{ github.actor }}
+                password: ${{ secrets.github_token }}
+        env:
+            GH_TOKEN: ${{ github.token }}
+
+        steps:
+            -   run: /usr/bin/supervisord -c /etc/supervisord.conf &
+
+            -   name: Checkout SwagPaymentPayPalUnified
+                uses: actions/checkout@v3
+                with:
+                    path: plugin
+
+            -   name: Move plugin
+                run: mv "$(pwd)/plugin" /shopware/custom/plugins/SwagPaymentPayPalUnified
+
+            -   name: Setup SwagPaymentPayPalUnified
+                run: |
+                    cd /shopware/custom/plugins/SwagPaymentPayPalUnified
+                    make init-legacy
+
+            -   name: Setup legacy tests
+                run: |
+                    cd /shopware/custom/plugins/SwagPaymentPayPalUnified
+                    composer require --dev phpcompatibility/php-compatibility
+
+            -   name: Run legacy tests
+                run: |
+                    cd /shopware/custom/plugins/SwagPaymentPayPalUnified
+                    ./vendor/bin/phpcs --config-set installed_paths vendor/phpcompatibility/php-compatibility
+                    make fix-cs-dry
+                    php ../../../bin/console -e production -q
```

Comment:
as PHPStan is not needed here, this should also be not needed. but I am not sure about the `PHPCompatibility` stuff